bwresources.py

```diff
@@ -639,15 +526,17 @@ def _name_to_id(self, attribute, setting):
                 ids.append(self.categories.ids[parent]["children"][child])
             return ids
 
-        elif attribute in ["parentCategory", "xparentCategory"]:
+        elif attribute in ["parentCategory", "xparentCategory", "parentCategories", "categories"]:
+            #plural included for get_charts syntax
```

Comment:
At some point we should find a way to make `BWQueries` and `BWGroups` use the same `_name_to_id` function (to make this thing easier to maintain).  That doesn't need to be in the scope of this PR, though.